### DISCUSSION TITLE
[Enhancement] [button] [Feature Request] cancel focus-style after clicked button

### DISCUSSION THREAD

**Discussion by michaelxuzhi** [element-plus/element-plus#8473] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `button`

## Description

after clicked a single button, it didn't reset its focus-style to default.I have to click another component or some empty place to cancel(reset) the button's style.

### Additional comments

i have fundamentally solved this problem from ,but i don't know how to commit it(pr).

**Comment by sxzz** [comment]
https://github.com/element-plus/element-plus/issues/8390#issuecomment-1163344732

**Reply by michaelxuzhi** [reply]
ok，谢谢你噢，看来是需要自行处理样式了，我还想着在button.vue中的handleClick事件下直接处理。
